src/metabase/query_processor/streaming/common.clj

```diff
@@ -69,7 +70,7 @@
   "Merge format settings defined in the localization preferences into the format settings
   for a single column."
   [format-settings global-settings-key]
-  (let [global-settings (global-settings-key (public-settings/custom-formatting))
+  (let [global-settings (get (public-settings/custom-formatting) global-settings-key)
```

Comment:
This is just curiosity: why is `(get {} :key)` preferred here over `(:key {})`? Just style pref? Something more consequential?

Reply:
Normally I just use the key as a function, but since the key is dereferenced value I felt that this gives some level of self-documentation -- it's obvious this is a lookup and not a ns-level function. I don't have a strong opinion on reverting this line, though.

Reply:
No, I think keep it. I was really just curious what the thought was. Thanks for the explanation.